Ensure we are Monitoring Instance Memory

emileswarts opened the pull request:
Cloudwatch doesn't monitor instance memory by default.
We need to pull in a custom script and add it to a cron (which will come
in a future PR). Ensure that this is present in the /scripts directory
for now.

emileswarts committed:
Ensure we are Monitoring Instance Memory

Cloudwatch doesn't monitor instance memory by default.
We need to pull in a custom script and add it to a cron (which will come
in a future PR). Ensure that this is present in the /scripts directory
for now.

adrianclay commented on tasks/cloudwatch_memory_monitoring.yml:
```diff
@@ -0,0 +1,17 @@
+    state: directory
+
+- name: Download Amazon CloudWatch Monitoring Scripts
+  get_url: url=http://aws-cloudwatch.s3.amazonaws.com/downloads/CloudWatchMonitoringScripts-1.2.2.zip dest={{ user_scripts_dir }}/CloudWatchMonitoringScripts-1.2.2.zip
```
HTTP 😿  Is there an HTTPS version?

emileswarts committed:
Download CloudWatch memory monitoring scripts over https

adrianclay commented on playbooks/frontend.yml:
```diff
@@ -23,3 +24,12 @@
+      import_tasks: '../tasks/cloudwatch_memory_monitoring.yml'
+      vars:
+        user_scripts_dir: "{{ scripts_dir }}"
+      when: scripts_dir_check.stat.exists == False
```
Would it not make more sense to be checking the `CloudWatchMonitoringScripts-1.2.2` directory as well?  In the case that the scripts dir is created by something else?

emileswarts committed:
Improve status check for presence of CloudWatch memory logging directory

Don't just check that the scripts directory exists, but the actual
CloudWatch memory script directory within it before creating or moving
on.
